components-core/src/main/java/com/adyen/checkout/components/core/internal/ui/model/CommonComponentParamsMapper.kt

```diff
@@ -23,7 +23,7 @@ class CommonComponentParamsMapper {
     ): CommonComponentParamsMapperData {
         val sessionParams: SessionParams? = dropInOverrideParams?.sessionParams ?: componentSessionParams
         val commonComponentParams = CommonComponentParams(
-            shopperLocale = checkoutConfiguration.shopperLocale ?: deviceLocale,
+            shopperLocale = checkoutConfiguration.shopperLocale ?: sessionParams?.shopperLocale ?: deviceLocale,
```

Comment:
Yes I had some discussions and there are 2 main reasons for this decision:
- If you are using standalone components you might not need to send a shopper locale to the sessions endpoint, if there is nothing to be localized on the API
- Overriding the shopper locale on the front end does not break the payment flow, unlike the rest of the session fields. So even though we cannot think of a concrete use case for this, it doesn't hurt to allow it especially since it's optional now

WDYT?

Reply:
Makes sense, just wanted to make sure this is intentional